### DISCUSSION TITLE
Issues Encountered While Following the Tutorial about triaxial DEM test

### DISCUSSION THREAD

**Discussion by ABF-cell** [GrainLearning/grainLearning#79] [Q&A]
While following the tutorial about triaxial DEM test, I encountered an error during the execution of the code. The  issue I faced is **"raise np.linalg.LinAlgError(msg) numpy.linalg.LinAlgError: When `allow_singular is False`, the input matrix must be symmetric positive definite**". If anyone else has encountered similar problems, or if you have suggestions for how to resolve this error, please feel free to share your insights. I’m currently stuck and any guidance would be greatly appreciated.

**Comment by chyalexcheng** [comment]
Hi, sorry for seeing this too late. Generally, this error is caused by a zero covariance matrix. Do you have a zero in your `calibration.system.obs_data`? I would suggest setting `calibration.inference.Bayes_filter.scale_cov_with_max = True`. Please note there has been a recent update in the repo. We have been working towards a new release.
